docs/diagrams/CommitActivityDiagram.puml

```diff
@@ -8,10 +8,10 @@ start
 'Since the beta syntax does not support placing the condition outside the
 'diamond we place it as the true branch instead.
 
-if () then ([command commits AddressBook])
+if () then ([command commits StaffBook])
     :Purge redundant states;
-    :Save AddressBook to
-    addressBookStateList;
+    :Save StaffBook to
+    StaffBookStateList;
```

Comment:
nit: should "staff" in "StaffBookStateList" not be capitalized?

Reply:
Fixed